src/main/java/com/sinthoras/visualprospecting/mixinplugin/MixinPlugin.java

```diff
@@ -93,6 +97,19 @@ private boolean loadJar(final String jarNameBeginsWith) {
         }
     }
 
+    public static File findJarOf(final TargetedMod mod) {
+        try {
+            return walk(MODS_DIRECTORY_PATH)
```

Comment:
Check also all sub-dirs. Also we drop the last dependency to timeconquerors mixins wrapper with this

Reply:
![image](https://user-images.githubusercontent.com/13066272/141834140-d124b776-8c79-443c-b4d8-5594f1ea0367.png)